### PR TITLE
"Learn by teaching" talk abstract

### PR DESCRIPTION
**Pull request by msvrtan**
Please help me abstract this talk: I would love to hear suggestions on the abstract/title but also would appreciate feedback on how interesting you feel the subject is.

I'm preparing a set of new talks for 2019, I'm aware I dont have the time to prepare all of them but as they are 'my babies' it's rather hard to be objective on how interesting a particular topic is.

### CODE REVIEW

**Comment by zoka123** [comment]
I look forward to hearing this talk from you :)

I'd suggest you embrace the term "mentoring" instead of "teaching", as I believe in most cases you haven't had a lecture or repeating the same content all over again, but rather work **with** your apprentices and prepare them for the real world.

I've seen you mentoring others over smokes/coffee/beer, on meetups and conferences, road trips, and you're damn good at it 👍

**Comment by zoka123** [comment]
There is also this quote that perfectly aligns with your story:

>teaching someone else is the best way to learn

**Comment by pelshoff** [comment]
Imo, the abstract is too much about you and not enough about the attendee. I would move most of the stuff about you to your bio and use the abstract to tell: who should come to your talk, why they need to come and what they will take away from it.

**Comment by msvrtan** [comment]
@zoka123 I actually do have lectures/workshops as well :)

**Comment by msvrtan** [comment]
@zoka123 & @pelshoff : I've added version 2, thank you for the feedback, would love to hear your input on it too!

File: talks/learn-by-teaching.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#### Version 2 (After some Pim & Zoka suggestions)
+
+
+Have you ever felt like there is not much more to accoplish in your career? 
+-or-
+Have you ever felt there are no new challenges and your career is at the peak? 
+
+Do you feel like you received more from the community than you gave back?
+-or-
+Do you feel like you received more than you gave back and want to even the balance?
```
Comment (DamirSvrtan) [inline talks/learn-by-teaching.md]: This one seems to vague? Received more than you gave back from who? This one and the one above the 'or' sound very similar.

File: talks/learn-by-teaching.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+While you can always find challenges by switching languages or companies, speak on local meetup or conference, open source your work or help maintain existing ones, I went a different road: decided to become a teacher and a mentor. If you are interested in learning more on how and what, join me for this talk.
```
Comment (DamirSvrtan) [inline talks/learn-by-teaching.md]: I'd shorten this a bit:

While you can always find challenges by switching languages or companies, doing open source or speaking on conferences, I decided to go down a different road: to become a teacher and a mentor. If you are interested in learning more on how and what, join me for this talk.

**Comment by DamirSvrtan** [comment]
This one is definitely the most polished one and I'd love to hear it!

**Comment by msvrtan** [comment]
Thank you all for reviewing this abstract!

**Pull request merged**